.github/workflows/publish-cli.yml

```
@@ -136,7 +136,6 @@ jobs:
 
       # Computes SHA-256 checksum
       - name: SHA-256 checksum
-        working-directory: cli
         run: shasum -a 256 "./target/production/wallet${{ matrix.ext }}"
```

```suggestion
        run: shasum -a 256 "../target/production/wallet${{ matrix.ext }}"
```
Not sure but if we remove `working-directory: cli`, it should be that way ?